### PR TITLE
Add Omgevingsbeleid to codebases we work with

### LINKED ISSUE #181: Add Omgevingsbeleid to Codebases we work with
This codebase should be listed in Assessment.

### PR DESCRIPTION
**Pull request by Ainali**
Fixes #181

### CODE REVIEW

File: codebases/omgevingsbeleid.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    <div class="cb-maturity-detail-title cb-maturity-s1">
+      <h1>Digital Omgevingsbeleid</h1>
+    </div>
+    <div class="cb-maturity-stage cb-maturity-s1">Stage 1: <strong>Assessment</strong></div>
+  </div>
+
+  <div class="cb-detail-intro">
+    <div class="cb-detail-info">
+      <h3>What it is</h3>
+      <p>The Digital Omgevingsbeleid was built in response to the 2022 <a href="https://www.government.nl/topics/spatial-planning-and-infrastructure/revision-of-environment-planning-laws">Environment and Planning Act</a> (Omgevingswet) in the Netherlands. The Digital Omgevingsbeleid lets you explore laws and policies, view where they apply and compare them to previous versions. This way, policies are more informative and up-to-date.</p>
```
Comment (ElenaFdR) [inline codebases/omgevingsbeleid.html:25]:
```suggestion
      <p>Digital Omgevingsbeleid was built in response to the 2022 <a href="https://www.government.nl/topics/spatial-planning-and-infrastructure/revision-of-environment-planning-laws">Environment and Planning Act</a> (Omgevingswet) in the Netherlands. Digital Omgevingsbeleid lets you explore laws and policies, view where they apply and compare them to previous versions. This way, policies are more informative and up-to-date.</p>
```

File: codebases/omgevingsbeleid.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      <h1>Digital Omgevingsbeleid</h1>
+    </div>
+    <div class="cb-maturity-stage cb-maturity-s1">Stage 1: <strong>Assessment</strong></div>
+  </div>
+
+  <div class="cb-detail-intro">
+    <div class="cb-detail-info">
+      <h3>What it is</h3>
+      <p>Digital Omgevingsbeleid was built in response to the 2022 <a href="https://www.government.nl/topics/spatial-planning-and-infrastructure/revision-of-environment-planning-laws">Environment and Planning Act</a> (Omgevingswet) in the Netherlands. Digital Omgevingsbeleid lets you explore laws and policies, view where they apply and compare them to previous versions. This way, policies are more informative and up-to-date.</p>
+      <p>For example, if someone wants to build a windfarm, they can quickly understand what's allowed on their prospective site. The Digital Omgevingsbeleid also makes it easier for a civil servant considering new regulation to see how the new regulations will interact with regulations that already apply.</p>
```
Comment (ElenaFdR) [inline codebases/omgevingsbeleid.html:26]:
```suggestion
      <p>For example, if someone wants to build a windfarm, they can quickly understand what's allowed on their prospective site. Digital Omgevingsbeleid also makes it easier for a civil servant considering new regulation to see how the new regulations will interact with regulations that already apply.</p>
```

File: codebases/omgevingsbeleid.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>How we're helping</h3>
+    <p>We are helping the Omgevingsbeleid team by:</p>
+    <ul>
+      <li>giving advice on open sources licenses</li>
```
Comment (ElenaFdR) [inline codebases/omgevingsbeleid.html:51]:
```suggestion
      <li>giving advice on open source licenses</li>
```

File: codebases/omgevingsbeleid.html [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+      <li>preparing for publishing the code in the open</li>
```
Comment (ElenaFdR) [inline codebases/omgevingsbeleid.html:52]:
```suggestion
      <li>helping them prepare to publish the code in the open</li>
```

File: codebases/omgevingsbeleid.html [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+    <div class="cb-maturity-detail-title cb-maturity-s1">
+      <h1>Digital Omgevingsbeleid</h1>
+    </div>
+    <div class="cb-maturity-stage cb-maturity-s1">Stage 1: <strong>Assessment</strong></div>
+  </div>
+
+  <div class="cb-detail-intro">
+    <div class="cb-detail-info">
+      <h3>What it is</h3>
+      <p>Digital Omgevingsbeleid was built in response to the 2022 <a href="https://www.government.nl/topics/spatial-planning-and-infrastructure/revision-of-environment-planning-laws">Environment and Planning Act</a> (Omgevingswet) in the Netherlands. Digital Omgevingsbeleid lets you explore laws and policies, view where they apply and compare them to previous versions. This way, policies are more informative and up-to-date.</p>
```
Comment (ElenaFdR) [inline codebases/omgevingsbeleid.html:25]:
```suggestion
      <p>Digital Omgevingsbeleid was built in response to the 2022 <a href="https://www.government.nl/topics/spatial-planning-and-infrastructure/revision-of-environment-planning-laws">Environment and Planning Act</a> (Omgevingswet) in the Netherlands. Digital Omgevingsbeleid lets you explore laws and policies, view where they apply and compare them to previous versions. This way, policies are more informative and up to date.</p>
```

Comment (ElenaFdR) [reply codebases/omgevingsbeleid.html]: "informative and up to date" isn't hugely useful - something like "easier to understand and always up to date" may make it clearer why this is such a valuable codebase.

File: codebases/omgevingsbeleid.html (hunk 1 above)
Comment (ElenaFdR) [reply codebases/omgevingsbeleid.html]: @Ainali, why did you reject the date move?

File: codebases/omgevingsbeleid.html [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+</div>
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>Community</h3>
+    <dl>
+      <dt>Created by:</dt>
+      <dd>the province of South Holland</dd>
```
Comment (ElenaFdR) [inline codebases/omgevingsbeleid.html:77]:
```suggestion
      <dd>Province of South Holland</dd>
```

File: codebases/omgevingsbeleid.html (hunk 1 above)
Comment (Ainali) [reply codebases/omgevingsbeleid.html]: I didn't reject it, I committed all your suggestions as they came up, not sure what GitHub is up to.

File: codebases/omgevingsbeleid.html [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+<hr>
+
+<div class="cb-section">
+  <div class="cb-detail-info">
+    <h3>How we're helping</h3>
+    <p>We are helping the Omgevingsbeleid team by:</p>
+    <ul>
+      <li>giving advice on open source licenses</li>
+      <li>helping them prepare to publish the code in the open</li>
```
Comment (ElenaFdR) [inline codebases/omgevingsbeleid.html:52]: Would this bullet from the March newsletter be more clear and specific than this line about preparing?
- "designing workshops on essential skills for open technical and product management of a public codebase."

Comment (Ainali) [reply codebases/omgevingsbeleid.html:52]: Perhaps, but we have just postponed that one (with an uncertain future).

File: codebases/omgevingsbeleid.html (hunk 1 above)
Comment (ElenaFdR) [reply codebases/omgevingsbeleid.html]: Bizarre.

Comment (Ainali) [reply codebases/omgevingsbeleid.html]: Changed it manually now

**Review by bvhme — APPROVED** [review]

**Review by ElenaFdR — APPROVED** [review]
Super snazzy!

**Pull request merged**